test-runner-frontend/src/Components/TestRunsTable.js

```diff
@@ -48,7 +48,13 @@ const columns = [
   },
 ];
 
-export default function TestRunsTable({ onCheckboxChange, isCheckBoxActive, testRunResults, sortDate }) {
+export default function TestRunsTable({
+  onCheckboxChange,
+  checkBoxSelection,
```

Comment:
is this being used anywhere?

Reply:
No, we replaced that with onCheckboxChange function, I removed it already.